crates/subspace-farmer/src/farm.rs

```diff
@@ -194,11 +194,11 @@ pub trait PieceCache: Send + Sync + fmt::Debug {
 /// Result of piece storing check
 #[derive(Debug, Copy, Clone, Encode, Decode)]
 pub enum MaybePieceStoredResult {
-    /// Definitely not stored
+    /// Piece is not stored, or it can't be added to the cache
     No,
```

Comment:
> or it can't be added to the cache
isn't piece not stored meant that it cannot be added to the cache ?

Reply:
Correct, it can't be

Reply:
This variant means both: the piece isn't stored already, and it definitely can't be added to the cache. There are other variants for "might be added", and "already in the cache", which is why this wording is needed.

If you have a suggestion for better wording, go for it!